docs/how-we-work.md

```diff
@@ -0,0 +1,131 @@
+
+### Minor or cosmetic
+
+This includes work like typos in comments or documentation, localized work, or rote mechanical changes.
+
+##### Examples
+
+* Changing or upgrading dependencies.
+* Renaming components.
+* Minor refactoring that only touches a file or two.
+
+##### Process
+
+1. Isolate work on a feature branch in the `atom/github` repository and open a pull request. Title-only pull requests are fine. If it's _really_ minor, like a one-line diff, committing directly to `master` is also perfectly acceptable.
+2. Ensure that our CI remains green across platforms.
+3. Merge your own pull request; no code review necessary.
+
+### Bug fixes
+
+Addressing unhandled exceptions, lock-ups, or correcting other unintended behavior in established functionality follows this process. For bug fixes that have UX, substantial UI, or package scope implications or tradeoffs, consider following [the new feature RFC process](#new-features) instead, to ensure we have a chance to collect design and community feedback before we proceed with a fix.
+
+##### Process
+
+1. Open an issue on `atom/github` describing the bug if there isn't one already.
+2. Identify the root cause of the bug and leave a description of it as an issue comment. If necessary, modify the issue body and title to clarify the bug as you go.
+3. When you're ready to begin writing the fix, assign the issue to yourself and move it to the "in progress" column on the [short-term roadmap project](https://github.com/atom/github/projects/8). :rainbow: _This signals to the team and to the community that it's actively being addressed, and keeps us from colliding._
+4. Work on a feature branch in the `atom/github` repository and open a pull request.
+5. Write a failing test case that demonstrates the bug (or a rationale for why it isn't worth it -- but bias toward writing one).
+6. Iteratively make whatever changes are necessary to make the test suite pass on that branch.
+7. Merge your own pull request and close the issue.
+8. _Depending on the severity of the bug:_ consider releasing a new version of the GitHub package and hot-fixing beta or stable with the fix. See ["How we ship"](#how-we-ship) for our criteria for each.
+
+### Re-architecting
+
+Major, cross-cutting refactoring efforts fit within this category. Our goals with work like this are to address technical debt, to improve our ability to comprehend the codebase, to ease the burdens of new contributors, and to make other planned features and bug fixes simpler to enact in the future.
+
+##### Examples
+
+* Porting the package from Etch to React.
+* Introducing additional diagnostics or metrics.
+* Restructuring an entire component tree.
+
+##### Process
+
+1. Propose these changes first in a conversation in Slack, a stand-up, or another synchronous channel. The decisions to be made here are:
+  * Does this change make sense to people other than me?
```

Comment:
Add at least one (the correct amount is two) space(s) here (and in the other similar level bullet points) in order for them to be properly indented when rendered.

Reply:
👍 Got it, thanks. I've also added a "rendered" link to the PR description too.